koku/masu/database/trino_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -190,9 +190,9 @@ SELECT cast(uuid() as varchar) as uuid,
             THEN  split_part(unitofmeasure, ' ', 2)
         ELSE unitofmeasure
     END) as unit_of_measure,
-    sum(coalesce(nullif(azure.quantity, 0), azure.usagequantity)) as usage_quantity,
+    sum(coalesce(nullif(azure.quantity, 0), 0)) as quantity,
```

Comment:
All of these `coalesce(nullif`s are for ensuring we select a quantity from either `azure.quantity` or `azure.usagequantity`. Since we are now only selecting from a single column, we can remove this crazy syntax.
```suggestion
    sum(azure.quantity) as usage_quantity,
```